funnel/views/index.py

```diff
@@ -38,6 +38,7 @@ class PolicyPage:
     PolicyPage('policy/refunds', __("Cancellation &amp; refund policy")),
     PolicyPage('policy/community', __("Community guidelines")),
     PolicyPage('policy/code', __("Code of conduct")),
+    PolicyPage('policy/posh', __("Prevention of Sexual Harassment Policy")),
```

Comment:
This should be in sentence case like the others. It _may_ also need to be abbreviated to "POSH policy" to fit the sidebar.